packages/dd-trace/src/profiling/profilers/events.js

```diff
@@ -377,28 +443,35 @@ function createPossionProcessSamplingFilter (samplingIntervalMillis) {
 class EventsProfiler {
   constructor (options = {}) {
     this.type = 'events'
-    this.eventSerializer = new EventSerializer()
+    this.maxSamples = getMaxSamples(options)
+    this.eventSerializer = new EventSerializer(this.maxSamples)
 
     const eventHandler = event => this.eventSerializer.addEvent(event)
     const eventFilter = options.timelineSamplingEnabled
       // options.samplingInterval comes in microseconds, we need millis
-      ? createPossionProcessSamplingFilter((options.samplingInterval ?? 1e6 / 99) / 1000)
+      ? createPoissonProcessSamplingFilter((options.samplingInterval ?? 1e6 / 99) / 1000)
```

Comment:
If I understand correctly, this PR changes `options.samplingInterval` to be milliseconds?

Reply:
Excellent catch, thanks!

Reply:
I'm now extra glad I made the "possion"->"poisson" typo fix as it drew your attention to this line 😁